### PR TITLE
add Dvorak, international with dead keys

### PR DESCRIPTION
**Pull request by alexsapps**
hi, i'm submitting dvorak international.  i just copied the programmer dvorak folder and did my best, using this page to get the layout name and description: http://www.tin.org/bin/man.cgi?section=7&topic=XKEYBOARD-CONFIG

what do you think about adding directions to the readme about how to add another layout to this repo?

### CODE REVIEW

**Comment by LaPingvino** [comment]
Only now I see that we seemingly did the same job :). Great to see another Dvorak International user :).

File: dvorakintl/manifest.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* Copyright 2014 Google Inc. All rights reserved.
```
Comment (bderrly) [inline dvorakintl/manifest.json]: How about updating the year to 2015 (the year you wrote this).

**Comment by alexsapps** [comment]
it's been a while and i'm out of the zone right now. anyone feel free to
copy my PR and make any needed modifications.

**Comment by LaPingvino** [comment]
I signed it before, but @alexsapps probably not 😄

**Pull request closed without merging**